src/main/java/core/basesyntax/service/operation/PurchaseOperation.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class PurchaseOperation implements OperationHandler {
+    private final FruitTransactionDao fruitTransactionDao;
+
+    public PurchaseOperation(FruitTransactionDao fruitTransactionDao) {
+        this.fruitTransactionDao = fruitTransactionDao;
+    }
+    @Override
+    public void recount(FruitTransaction transaction) {
```

Comment:
The `recount` method name is not descriptive of the operation being performed. Consider renaming it to something more specific to the purchase operation, like `processPurchase` or `executeTransaction`.